### PR TITLE
Build openstack-cheat-sheet page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build page on `/openstack/openstack-cheat-sheet`
- Update "OpenStack cheat sheet" button on `/openstack/tutorials` to point to `/openstack/openstack-cheat-sheet` 

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/openstack-cheat-sheet
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please check page against [copy doc](https://docs.google.com/document/d/1q37DrtjZhcgRK4NuL3CUsCv1aU6AQDX5uqtQ4ndVT-A/edit#)
- Please QA at https://ubuntu-com-11175.demos.haus/openstack/tutorials, https://ubuntu-com-11175.demos.haus/openstack/what-is-openstack and https://ubuntu-com-11175.demos.haus/openstack/openstack-cheat-sheet

## Issue / Card

Fixes [#4773](https://github.com/canonical-web-and-design/web-squad/issues/4773)

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)

### CODE REVIEW

File: templates/openstack/openstack-cheat-sheet.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{% block meta_description %}Instances (Nova), networks (Neutron), images (Glance), volumes (Cinder), identities (Keystone) and much more. Get started with OpenStack quickly.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1q37DrtjZhcgRK4NuL3CUsCv1aU6AQDX5uqtQ4ndVT-A/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>OpenStack Cheat Sheet</h1>
+      <p>The most comprehensive OpenStack cheat sheet for beginners and more advanced users.</p>
+      <p> What’s included:</p>
```
Comment (anthonydillon) [inline templates/openstack/openstack-cheat-sheet.html:13]: Can you straighten this quote please.

File: templates/openstack/openstack-cheat-sheet.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        <ul class="p-list">
+          <li class="p-list__item is-ticked"> Keystone </li>
+          <li class="p-list__item is-ticked"> Glance </li>
+          <li class="p-list__item is-ticked"> Neutron </li>
+          <li class="p-list__item is-ticked"> Nova </li>
+          <li class="p-list__item is-ticked"> Cinder </li>
+        </ul>
+      <p>Get the following OpenStack cheat sheet for free. No registration needed!</p>
+      <a href="https://assets.ubuntu.com/v1/8d3130a1-OpenStack.cheat.sheet.1.pdf?_ga=2.38929315.1481755299.1641543595-1851044449.1622191471" class="p-button--positive">Get OpenStack cheat sheet</a>
```
Comment (anthonydillon) [inline templates/openstack/openstack-cheat-sheet.html:22]: This is not good English.
```suggestion
      <a href="https://assets.ubuntu.com/v1/8d3130a1-OpenStack.cheat.sheet.1.pdf?_ga=2.38929315.1481755299.1641543595-1851044449.1622191471" class="p-button--positive">Get the OpenStack cheat sheet</a>
```

File: templates/openstack/openstack-cheat-sheet.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+{% block title %}The most comprehensive OpenStack cheat sheet{% endblock %}
+{% block meta_description %}Instances (Nova), networks (Neutron), images (Glance), volumes (Cinder), identities (Keystone) and much more. Get started with OpenStack quickly.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1q37DrtjZhcgRK4NuL3CUsCv1aU6AQDX5uqtQ4ndVT-A/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip--suru-topped">
+  <div class="row u-equal-height">
+    <div class="col-6">
+      <h1>OpenStack Cheat Sheet</h1>
```
Comment (anthonydillon) [inline templates/openstack/openstack-cheat-sheet.html:11]: This is not a name.
```suggestion
      <h1>OpenStack cheat sheet</h1>
```

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Free comments in the code but mostly the based on the copy

**Comment by minkyngkm** [comment]
Thanks @anthonydillon for reviewing! (It was actually WIP earlier) Hopefully it looks better now :)

**Comment by minkyngkm** [comment]
Hi @spencerbygraves! Could you please take a look at this page for a design review when you have time? :)

**Review by anthonydillon — CHANGES_REQUESTED** [review]
You should add this new page to the navigation of the site so the user doesn't lose there way

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Should the button be full width? 
image

File: navigation.yaml [hunk 4]
```diff
@@ -179,6 +179,8 @@ openstack:
       path: /openstack/docs
     - title: Tutorials
       path: /openstack/tutorials
+    - title: OpenStack Cheat sheet
```
Comment (anthonydillon) [inline navigation.yaml:182]:
```suggestion
    - title: OpenStack cheat sheet
```

**Comment by minkyngkm** [comment]
Thanks @anthonydillon for reviewing! I've made changes and also just moved the CTA to the left. 
One question - I was just wondering if it's better for the title of this page and url to be just "Cheat sheet" without "OpenStack" in the navigation. 
image

**Comment by minkyngkm** [comment]
The navigation title and url have been updated from "OpenStack cheat sheet to "Cheat sheet" 
image

**Comment by pmahnke** [comment]
@minkyngkm @davegoddard42 @sowasred2012 I have pinged marketing.  I find this page very strange on its own, not as a strip on another page. Please don't publish yet.

**Comment by pmahnke** [comment]
@minkyngkm I have a response:

> Hi @Peter Agree with your suggestion of having a strip on tutorials and what is openstack pages (+hidden in the menu for ranking purposes)

**Comment by pmahnke** [comment]
@minkyngkm I updated the /[tutorials](https://docs.google.com/document/d/1_4EJlX-fbcQBVC9Ojhjdbls6kbmITGvHKi5IEqw-HAA/edit#) and /[what-is-openstack](https://docs.google.com/document/d/15xrgIq_KUwHaCyFoI7-yS_JmBbsoyXJ4GpRMFAa7_is/edit#) pages.  

The strip can be an include as they are the same.

Please hide the cheat sheet page from the navigation.

You might want to revert its name back to /openstack-cheat-sheet for SEO.

**Comment by minkyngkm** [comment]
Thanks @pmahnke! I have made some updates.

**Comment by spencerbygraves** [comment]
@minkyngkm I think you could make the subheading h4, but otherwise it looks good.

Thanks.

**Comment by minkyngkm** [comment]
Thanks @spencerbygraves for your feedback! Its been updated! :)

image

**Comment by cristinadresch** [comment]
@pmahnke could you rereview this one please, thanks

**Review by pmahnke — APPROVED** [review]
+1, I think it looks good.

**Pull request merged**